sunpy/util/scraper.py

```diff
@@ -333,7 +355,7 @@ def _smallerPattern(self, directoryPattern):
         except Exception:
             raise
 
-    def _extract_files_meta(self, timerange, extractor=None):
+    def _extract_files_meta(self, timerange, extractor, matcher=False):
```

Comment:
```suggestion
    def _extract_files_meta(self, timerange, extractor, matcher=None):
```
I feel like if matcher is not a boolean, but a dictionary, `None` would fit better as default than `False`

Reply:
Or even an empty dictionary...

Reply:
Changed to `None`